client/src/components/DataFiles/DataFilesModals/DataFilesModals.scss

```
@@ -9,8 +9,10 @@
   .h-100 {
     padding: 0 10px;
   }
+  .breadcrumbs-container {
```

This does not apply, because it is misspelled.[^1]

[^1]: Though maybe it should be this name, because child is `.breadcrumbs`, but it's not yet. I would happily review a rename in a separate PR.

```suggestion
  .breadcrumb-container {
```